controllers/apply/building-connections/fields.js

```diff
@@ -49,54 +37,77 @@ const yourIdea = [
-                    <li>If applying for <strong>over £50,000</strong> please also tell us who will deliver the project and the experience they have</li>
+                helpText: `<ul>
+                    <li>Describe the project you would like funding for</li>
+                    <li>Tell us why it is needed in your area</li>
+                    <li>Describe who will deliver the project and the experience they have</li>
                 </ul>`
             }
         ]
     }
 ];
 
-const projectLocation = [
+const projectActivities = [
     {
-        legend: 'Where will your project take place?',
+        legend: 'Project activities, outcomes, and milestones',
         fields: [
             {
-                name: 'location',
-                type: 'checkbox',
-                options: PROJECT_LOCATIONS,
+                name: 'project-activities-outline',
+                type: 'textarea',
+                rows: 6,
                 isRequired: true,
-                label: 'Project region',
-                explanation: 'Select all regions that apply'
+                label: 'Briefly outline what your project is aiming to achieve'
             },
             {
-                name: 'project-location',
-                type: 'text',
-                size: 60,
+                name: 'project-activities-first-quarter',
+                type: 'textarea',
+                rows: 6,
+                isRequired: true,
+                label: 'What will be achieved in the first quarter Jan—March 2019?'
+            },
+            {
+                name: 'project-activities-first-year',
+                type: 'textarea',
+                rows: 6,
                 isRequired: true,
-                label: 'Project location',
+                label: 'What will be achieved in the financial year 2019/20?',
                 explanation:
-                    'In your own words, describe the locations that you’ll be running your project in. eg. “Newcastle community centre” or “Alfreton, Derby and Ripley”.'
+                    'If your project finishes before the end of the 2020 financial year include up until the end of your project.'
             }
         ]
     }
 ];
 
-const currentWork = [
+const projectLocation = [
     {
-        legend: 'Current work',
+        legend: 'Project location',
         fields: [
             {
-                name: 'current-work',
-                type: 'textarea',
-                rows: 14,
+                name: 'location',
+                type: 'checkbox',
+                options: [
+                    'North East',
+                    'North West',
+                    'Yorkshire and the Humber',
+                    'East Midlands',
+                    'West Midlands',
+                    'East of England',
+                    'Greater London',
+                    'South East',
+                    'South West'
+                ].map(_ => ({ label: _, value: _ })),
```

Comment:
This `.map` feels a little silly, might see if I can handle this logic in the form model, i.e. allow either an array (where labels + values are the same) or an object. Maybe overthinking this.

Reply:
Works for me really.